modules/gitops-configuring-argo-cd-oidc.adoc

```diff
@@ -0,0 +1,77 @@
+// Module is included in the following assemblies:
+//
+// * 
+
+[id="configuring-argo-cd-oidc_{context}"]
+= Configuring Argo CD OIDC
+
+.Prerequisites
+
+* You have obtained your client secret. 
+
+.Procedure
+
+. Store the client secret you generated earlier in the argocd secret argocd-secret.
+
+. Encode the client secret in base64:
++
+[source,terminal]
+----
+$ echo -n '83083958-8ec6-47b0-a411-a8c55381fbd2' | base64
+----
+
+. Edit the secret and add the base64 value to an `oidc.keycloak.clientSecret` key:
++
+[source,terminal]
+---- 
+$ kubectl edit secret argocd-secret -n argocd
+----
++
+.Example YAML of the secret
+----
+yaml apiVersion: v1 
+kind: Secret 
+metadata: name: argocd-secret 
+data:
+  oidc.keycloak.clientSecret: ODMwODM5NTgtOGVjNi00N2IwLWE0MTEtYThjNTUzODFmYmQy …
+----
+
+. Edit the `argocd` Custom Resource and add the OIDC configuration to enable the Keycloak authentication:
++
+[source,terminal]
+----
+$ kubectl edit argocd -n argocd
+----
++
+.Example of `argocd` Custom Resource:
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  creationTimestamp: null
+  name: argocd
+  namespace: argocd
+spec:
+  resourceExclusions: |
+    - apiGroups:
+      - tekton.dev
+      clusters:
+      - '*'
+      kinds:
+      - TaskRun
+      - PipelineRun
+  oidcConfig: |
+    name: OpenShift Single Sign-On
+    issuer: https://keycloak.example.com/auth/realms/myrelam <1>
```

Comment:
There's a typo here - `myrelam` should be `myrealm`

Reply:
Fixed.

Reply:
Fixed